ci/setup-gdk.ps1

```diff
@@ -0,0 +1,142 @@
+        "package", `
+        "retrieve", `
+        "tools", `
+        "schema_compiler-x86_64-win32", `
+        "$($pinned_core_sdk_version)", `
+        "$($core_sdk_dir)\tools\schema_compiler-x86_64-win32.zip" `
+    )
+    Start-Process -Wait -PassThru -NoNewWindow -FilePath "spatial" -ArgumentList @(`
+        "package", `
+        "retrieve", `
+        "schema", `
+        "standard_library", `
+        "$($pinned_core_sdk_version)", `
+        "$($core_sdk_dir)\schema\standard_library.zip" `
+    )
+    Start-Process -Wait -PassThru -NoNewWindow -FilePath "spatial" -ArgumentList @(`
+        "package", `
+        "retrieve", `
+        "worker_sdk", `
+        "c-dynamic-x86-msvc_md-win32", `
+        "$($pinned_core_sdk_version)", `
+        "$($core_sdk_dir)\worker_sdk\c-dynamic-x86-msvc_md-win32.zip" `
+    )
+
+    Start-Process -Wait -PassThru -NoNewWindow -FilePath "spatial" -ArgumentList @(`
+        "package", `
+        "retrieve", `
+        "worker_sdk", `
+        "c-dynamic-x86_64-msvc_md-win32", `
+        "$($pinned_core_sdk_version)", `
+        "$($core_sdk_dir)\worker_sdk\c-dynamic-x86_64-msvc_md-win32.zip" `
+    )
+    Start-Process -Wait -PassThru -NoNewWindow -FilePath "spatial" -ArgumentList @(`
+        "package", `
+        "retrieve", `
+        "worker_sdk", `
+        "c-dynamic-x86_64-gcc_libstdcpp-linux", `
+        "$($pinned_core_sdk_version)", `
+        "$($core_sdk_dir)\worker_sdk\c-dynamic-x86_64-gcc_libstdcpp-linux.zip" `
+    )
+    Finish-Event "download-spatial-packages" "build-unreal-gdk-:windows:"
+
+
+    Start-Event "extract-spatial-packages" "build-unreal-gdk-:windows:"
+    Expand-Archive -Path "$($core_sdk_dir)\tools\schema_compiler-x86_64-win32.zip" -DestinationPath "$($binaries_dir)\Programs\" -Force
+    Expand-Archive -Path "$($core_sdk_dir)\schema\standard_library.zip" -DestinationPath "$($binaries_dir)\Programs\schema\" -Force
+    Expand-Archive -Path "$($core_sdk_dir)\worker_sdk\c-dynamic-x86-msvc_md-win32.zip" -DestinationPath "$($binaries_dir)\Win32\" -Force
+    Expand-Archive -Path "$($core_sdk_dir)\worker_sdk\c-dynamic-x86_64-msvc_md-win32.zip" -DestinationPath "$($binaries_dir)\Win64\" -Force
+    Expand-Archive -Path "$($core_sdk_dir)\worker_sdk\c-dynamic-x86_64-gcc_libstdcpp-linux.zip" -DestinationPath "$($binaries_dir)\Linux\" -Force
+    Finish-Event "extract-spatial-packages" "build-unreal-gdk-:windows:"
+
+
+    # Copy from binaries_dir
+    Copy-Item "$($binaries_dir)\Win64\include\*" "$($worker_sdk_dir)\" -Force -Recurse
+
+    Start-Event "build-utilities" "build-unreal-gdk-:windows:"
+    $msbuild_proc = Start-Process -PassThru -NoNewWindow -FilePath "$($msbuild_exe)" -ArgumentList @(`
+        "/nologo", `
+        "SpatialGDK\Build\Programs\Improbable.Unreal.Scripts\Improbable.Unreal.Scripts.sln", `
+        "/property:Configuration=Release" `
+    )
+
+    # Working around a powershell bug
```

Comment:
Was there a jira ticket for this? If so please link the number

Reply:
This actually doesn't seem to be a bug per se - I think it's expected behaviour with PS. Have updated the comment to reflect this better